README.md

```diff
@@ -27,7 +29,23 @@ steps:
 - run: julia -e 'println("Hello, World!")'
 ```
 
-### Matrix Testing:
+### Julia Versions
+
+You can either specify specific Julia versions or version ranges. If you specify a version range, the **highest** available Julia version that matches the range will be selected.
+
+#### Examples
+
+- `1.2.0` is a valid semver version. The action will try to download exactly this version. If it's not available, the build step will fail.
+- `1.0.x` is a version range that will match the highest available Julia version that starts with `1.0`, e.g. `1.0.5`.
```

Comment:
Wouldn't it be better if one specified this case as `1.0`, rather than `1.0.x`? I think that would be more in line with the other CI platforms.

Reply:
Just remembered why I chose `1.0.x`. The yaml parsers parses `1.0` as `1`, so we have to wrap `1.0` in quotes...

Reply:
Ah, that is annoying!